pkg/machines/libvirt.es6

```diff
@@ -476,8 +516,51 @@ LIBVIRT_PROVIDER = {
             });
         };
     },
+    GET_HYPERVISOR_MAX_VCPU ({ connectionName }) {
+        logDebug(`${this.name}.GET_HYPERVISOR_MAX_VCPU:`);
+        if (connectionName) {
+            return dispatch => spawnVirsh({connectionName,
+                method: 'GET_HYPERVISOR_MAX_VCPU',
+                failHandler: buildFailHandler({ dispatch, connectionName, message: _("GET HYPERVISOR MAX VCPU action failed")}),
+                args: ['maxvcpus']
+            }).then((count) => dispatch(setHypervisorMaxVCPU(count)));
```

Comment:
If `GET_HYPERVISOR_MAX_VCPU` is connection-specific, it should be reflected in redux storage same as in the dialog.

Please note, there are recently `system` and `session` connections, plus "logical" for provider (like oVirt) and in plan is a generic connection URI https://github.com/cockpit-project/cockpit/issues/7484 .